src/main/java/fi/helsinki/cs/tmc/cli/command/OrganizationCommand.java

```diff
@@ -0,0 +1,118 @@
+package fi.helsinki.cs.tmc.cli.command;
+
+import fi.helsinki.cs.tmc.cli.backend.Account;
+import fi.helsinki.cs.tmc.cli.backend.TmcUtil;
+import fi.helsinki.cs.tmc.cli.core.AbstractCommand;
+import fi.helsinki.cs.tmc.cli.core.CliContext;
+import fi.helsinki.cs.tmc.cli.core.Command;
+import fi.helsinki.cs.tmc.cli.io.EnvironmentUtil;
+import fi.helsinki.cs.tmc.cli.io.Io;
+import fi.helsinki.cs.tmc.cli.utils.OptionalToGoptional;
+import fi.helsinki.cs.tmc.core.domain.Organization;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Options;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import com.google.common.base.Optional;
+
+@Command(name = "organization", desc = "Change organization")
+public class OrganizationCommand extends AbstractCommand {
+
+    private CliContext ctx;
+    private Io io;
+
+    @Override
+    public void getOptions(Options options) {
+        options.addOption("o", "organization", true, "Slug of organization");
+    }
+
+    @Override
+    public void run(CliContext ctx, CommandLine args) {
+        Optional<Organization> organization = chooseOrganization(ctx, args);
+        Account currentAccount = this.ctx.getSettings().getAccount();
+        currentAccount.setOrganization(organization);
+        this.ctx.getSettings().setAccount(currentAccount);
+    }
+
+    private List<Organization> listOrganizations() {
+        if (!ctx.loadBackend()) {
+            return null;
+        }
+        List<Organization> organizations = TmcUtil.getOrganizationsFromServer(ctx);
+        return organizations;
+    }
+
+    private void printOrganizations(List<Organization> organizations, boolean pager) {
+        if (organizations.isEmpty()) {
+            io.print("No organizations found.");
+            return;
+        }
+        List<Organization> pinned = new ArrayList();
+        List<Organization> others = new ArrayList();
+        organizations.stream()
+                .sorted(Comparator.comparing(Organization::getName))
+                .forEach(o -> {
+                    if (o.isPinned()) {
+                        pinned.add(o);
+                    } else {
+                        others.add(o);
+                    }
+                });
+        System.out.println();
```

Comment:
Some parts of the code use `System.println` for printing and other parts `io.println`.